src/noos_inv/tasks/python.py

```diff
@@ -146,7 +146,7 @@ def release(
         case types.InstallType.PIPENV:
             ctx.run(f"pipenv run twine upload dist/* -u {user} -p {token}", pty=True)
         case types.InstallType.UV:
-            ctx.run(f"uvx twine upload dist/* -u {user} -p {token}")
+            ctx.run(f"uv build && uv publish -u {user} -p {token}", pty=True)
```

Comment:
If we are using tokens, then `user` should always be `__token__`, in that case `uv` provides a shortcut:
```suggestion
            ctx.run(f"uv build && uv publish -t {token}", pty=True)
```